acceptance/acceptance_suite_test.go

```diff
@@ -453,22 +453,22 @@ func checkDependencies() error {
 }
 
 func FailWithReport(message string, callerSkip ...int) {
-	fmt.Println("\nA test failed. You may find the following information useful for debugging:")
-	fmt.Println("The cluster pods: ")
-	out, err := helpers.Kubectl("get pods --all-namespaces")
-	if err != nil {
-		fmt.Print(err.Error())
-	} else {
-		fmt.Print(out)
-	}
-
-	fmt.Println("The cluster deployments: ")
-	out, err = helpers.Kubectl("get deployments --all-namespaces")
-	if err != nil {
-		fmt.Print(err.Error())
-	} else {
-		fmt.Print(out)
-	}
+	// fmt.Println("\nA test failed. You may find the following information useful for debugging:")
```

Comment:
I accidentally committed these changes. On the other hand, this output hasn't been very helpful lately, should we maybe remove it? I can keep the method as a placeholder for local debugging if we need to add debugging output and such.